packages/logger/README.md

```diff
@@ -34,7 +36,7 @@ import { logger } from '@tinyhttp/logger'
 const app = new App()
 
 app
-  .use(logger({ methods: ['GET', 'POST'], timestamp: { format: 'HH:mm:ss' } }))
+  .use(logger({ methods: ['GET', 'POST'], timestamp: { format: 'HH:mm:ss' }, outputConfiguration: { output: console.log, color: false } }))
```

Comment:
What about changing `outputConfiguration` to `output`, and `output` to `function`?

Reply:
I thought `callback` might me better, but I think both are ambiguous.

Reply:
@Arnovsky hmm `callback` sounds good